src/App.php

```diff
@@ -182,7 +182,11 @@ public function getMode()
 	 */
 	public function registerStylesheet($path)
 	{
-		$url = str_replace($this->getBasePath() . DIRECTORY_SEPARATOR, '', $path);
+		if (!empty($this->getBasePath())) {
+			$url = str_replace($this->getBasePath() . DIRECTORY_SEPARATOR, '', $path);
```

Comment:
`str_replace` was never the right function for this. Instead, consider the following code:
```
if (mb_strpos($path, $this->getBasePath() . DIRECTORY_SEPARATOR) == 0) {
	$path = mb_substr($path, mb_strlen($this->getBasePath() . DIRECTORY_SEPARATOR));
}
```